packages/office-ui-fabric-react/src/components/Dialog/DialogContent.tsx

```diff
@@ -35,10 +35,11 @@ export class DialogContent extends BaseComponent<IDialogContentProps, IDialogCon
   public render() {
     let {
       showCloseButton,
-      closeButtonAriaLabel,
+      closeButtonAriaLabel = 'Close',
```

Comment:
we should add it to defaultprops defined above instead of render() method that will keep all defaults at single place.